health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -54,6 +54,52 @@ export const UICustomizations = {
+      data.body.BoundaryTypeHierarchySearchCriteria.hierarchyType = data?.state?.searchForm?.Name;
+      console.log("final data is", data);
+      return data;
+    },
+    additionalCustomizations: (row, key, column, value, t, searchResult) => {
+      console.log("the row is", row);
+      const [isActive, setIsActive] = useState(row?.isActive);
+        switch (key) {
+          case "Hierarchy Name":
+            console.log("hoerarchy name", row?.hierarchyType);
+            return row?.hierarchyType;
+            break;
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)